specification/resources/resource-manager/readme.go.md

```diff
@@ -109,6 +110,16 @@ namespace: policy
 output-folder: $(go-sdk-folder)/services/resources/mgmt/2019-09-01/policy
 ```
 
+### Tag: package-policy-2020-09 and go
+
+These settings apply only when `--tag=package-policy-2020-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-policy-2020-09' && $(go)
+namespace: policy
+output-folder: $(go-sdk-folder)/services/resources/mgmt/2020-09-01/policy
+```
```

Comment:
We need to remove this piece of config, since it duplicates the settings above (L103-L111)

Reply:
Its a different tag and output-folder, 2020-09 and 2019-09, it was just out of order.  I will fix the order to make it clearer.  There isn't an actual issue here right?

Reply:
Changing the ordering causes GO generation to fail with some sort of ChangeLog related error... I just moved 2020-09 up above 2019-09 instead.  Is GO generation really dependent on the ordering of these declarations in the readme?

Reply:
GO is now complaining about an old changelog which seems new and not related to the ordering change in the readme

Reply:
Ah - that is a known issue and has been resolved.

Reply:
And you are right, I might be dizzy then....